README.MD

```diff
@@ -67,6 +67,37 @@ The [design document](https://microsoft.github.io/presidio/design.html) introduc
 2. Create a Presidio project
 3. Start using the Presidio analyze and anonymize services
 
+## Single click deployment using the default values
+
+The script will install Presidio on your Kubenetes cluster. Prerequesites:
+1. Make sure you have a Kubernetes cluster.
+2. `kubectl` installed and verify you can communicate with the cluster by running: 
+```
+kubectl version
+```
+3. Local `helm` client.
+4. Recent presidio repo is cloned on your local machine.
+
+
+Navigate into `<root>\deployment` from command line.
+
+If You have helm installed, but havn't run `helm init`,
+then execute [deploy-helm.sh](deploy-helm.sh) in the command line. It will install `tiller` (helm server side) on your cluster.
+
+Make sure to [grant AKS access to ACR.](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks)
+
+If you already have `helm` and `tiller` configured, or if you installed it in the previous step, 
+execute [deploy-presidio.sh](deploy-presidio.sh) in the command line as follows:
+```sh
+deploy-presidio.sh <registry>
+```
+
+The script will install Presidio on your cluster using the deault values. 
```

Comment:
```suggestion
The script will install Presidio on your cluster using the default values. 
```

Reply:
Also, what happens during an upgrade? I really don't see the value in the bash script to install since there is no logic inside, just helm install which you have to configure anyway with version numbers.

Reply:
the idea is to supply a one-click deployment script...

Reply:
yes but it doesn't make since you replace one helm chart command with bash script

Reply:
the script is not for upgrade. just first time installation. @itye-msft , please provide input on this issue.